tests/amqpvalue_ut/amqpvalue_ut.c

```diff
@@ -210,7 +210,7 @@ TEST_FUNCTION(amqpvalue_create_null_succeeds)
 {
     // arrange
 	AMQP_VALUE result;
-    EXPECTED_CALL(gballoc_malloc(IGNORED_NUM_ARG));
+    STRICT_EXPECTED_CALL(gballoc_malloc(IGNORED_NUM_ARG));
```

Comment:
At some point I decided I should only use STRICT_. Because otherwise it is much easier to err on the side of not validating a parameter. And the fact that IGNORE_NUM_ARG and IGNORE_PTR_ARG do not force writing an extra line makes it the same to use STRICT_EXPECTED_CALL or EXPECTED_CALL from the point of view of amount of code.